src/selector-native.js

```diff
@@ -121,6 +121,9 @@ jQuery.extend({
 
 		return results;
 	},
+	unique: function( results ) {
+		return this.uniqueSort( results );
```

Comment:
Since the "method" is actually context-free, I'd rather not introduce a reliance upon `this`. `jQuery.unique = jQuery.uniqueSort` or `var uniqueSort = function …; jQuery.extend({ …, unique: uniqueSort, uniqueSort: uniqueSort, … })` would be better. Other than that, though, LGTM!